src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/UIElements.java

```diff
@@ -0,0 +1,105 @@
+package com.qaprosoft.carina.demo.mobile.gui.pages.android;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.demo.mobile.gui.pages.common.UIElementsBase;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.AndroidElement;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.events.EventFiringWebDriver;
+
+import static io.appium.java_client.touch.offset.PointOption.point;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = UIElementsBase.class)
+public class UIElements extends UIElementsBase {
+    @FindBy(id = "editText")
+    private ExtendedWebElement text;
+
+    @FindBy(id = "editText2")
+    private ExtendedWebElement email;
+
+    @FindBy(id = "editText3")
+    private ExtendedWebElement date;
+
+
+    @FindBy(id = "checkBox2")
+    private ExtendedWebElement checkBox;
+
+    @FindBy(id = "radioButton")
+    private ExtendedWebElement maleRadioButton;
+
+    @FindBy(id = "radioButton3")
+    private ExtendedWebElement femaleRadioButton;
+
+    @FindBy(id = "radioButton5")
+    private ExtendedWebElement otherRadioButton;
+
+    @FindBy(id = "radioButton5")
+    private ExtendedWebElement seekBar;
+
+
+    public void fillInText(String text) {
+        this.text.type(text);
+    }
+
+    public void fillInEmail(String email) {
```

Comment:
- Please rename it from "fillInText" to "typeTextFld"
- There is no sense to use "this" here